wezterm-input-types/src/lib.rs

```diff
@@ -1975,7 +1975,8 @@ bitflags! {
         // so that we effective have Option<bool>
         const MACOS_FORCE_DISABLE_SHADOW = 4;
         const MACOS_FORCE_ENABLE_SHADOW = 4|8;
-        const INTEGRATED_BUTTONS = 16;
+        const MACOS_FORCE_SQUARE_CORNERS = 16;
+        const INTEGRATED_BUTTONS = 32;
```

Comment:
prefer to minimize the diff unless there is good reason to re-number the enum

```suggestion
        const INTEGRATED_BUTTONS = 16;
        const MACOS_FORCE_SQUARE_CORNERS = 32;
```